src/containers/ProjectsList.js

```diff
@@ -22,19 +23,27 @@ export class ProjectsList extends Component {
     filteredProjects: {},
     totalProjects: null,
     selectedLanguage: null,
-    languages: []
+    languages: [],
+    error: false
   };
 
-  componentDidMount () {
+  componentDidMount = async () => {
     if (!this.props.projects.length) {
-      this.props.fetchProjects()
+      await this.props.fetchProjects()
+      if (this.props.projects[0].error) {
```

Comment:
I think it would be  a good idea to put the error outside the projects, what I mean is putting it in the initial state file
`{
    projects: [],
   error: null || '',
    ....
 }`